lib/utils.ts

```diff
@@ -69,3 +73,7 @@ export function setPopupPath(path?: `/${string}`, cb: () => void = () => {}) {
 export function convertIPFStoHTTP(url: string) {
   return url.replace("ipfs://", "https://ipfs.io/ipfs/");
 }
+
+export async function getWalletSettings() {
```

Comment:
I didn't find anywhere use it. Is it okay to remove it?

Reply:
505d056bd370debd7350a26d0a26e8c5488a0afe